scripts/cnn_variant_wdl/README.md

```diff
@@ -0,0 +1,68 @@
+# gatk4-cnn-variant-filter
+
+### Purpose :
+This repo provides workflows that takes advantage of GATKs CNN tool which is a deep learning 
+approach to filter variants based on Convolutional Neural Networks. 
+
+Please read the following discussion to learn more about the CNN tool: [Deep Learning in GATK4](https://gatkforums.broadinstitute.org/gatk/discussion/10996/deep-learning-in-gatk4).
```

Comment:
This link still works, but is it going to move to the new website?

Reply:
I talked to Bhanu and the education team and they suggested to use a git repo where they uploaded all the old blog/discussion posts. Since this is a blog post and not a article, they plan on moving it to the new site later. 

https://github.com/broadinstitute/gatk-docs/blob/3333b5aacfd3c48a87b60047395e1febc98c21f9/blog-2012-to-2019/2017-12-21-Deep_learning_in_GATK4.md